src/dialects/snowflake/query-generator.d.ts

```diff
@@ -1,3 +1,5 @@
 import { AbstractQueryGenerator } from '../abstract/query-generator.js';
 
-export class SnowflakeQueryGenerator extends AbstractQueryGenerator {}
+export class SnowflakeQueryGenerator extends AbstractQueryGenerator {
+  showIndexesQuery(): string;
```

Comment:
Is this the correct way to overwrite the typings from `AbstractQueryGenerator` or does this now allow both?

Reply:
Not sure you can change the signature

Does snowflake even implement `showIndexesQuery`? Its current implementation is `SELECT '' FROM DUAL` which is very strange.

Looks like not: https://dataedo.com/kb/query/snowflake/list-unique-keys-and-indexes-in-the-database

This is more a snowflake implem problem than a typing issue

Reply:
Yeah, I found the implementation strange but there is no way for me to really test it so I didn't doubt it for now. I can add a TODO to the implementation? And I'll just remove this signature then

Reply:
Sure, TODO is fine. Especially since we can't test it